drivers/scheduler/dcos/mesos_ops.go

```diff
@@ -0,0 +1,108 @@
+package dcos
```

Comment:
Why don't we start with moving this package under the new portworx/sched-ops repo now itself? Instead of doing it later.

Reply:
Also the marathon sched ops

Reply:
I wanted to get basic dcos tests running before I move all these ops in the other repo, as I am actively modifying the ops. It will be a hassle to keep maintaining the sched-ops up to date with every change I make here. Will migrate is soon after I have the tests passing.